.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+repos:
+
+# Check if the commit message has a valid sign-off.
+- repo: https://github.com/gklein/check_signoff
+  rev: v1.0.5
+  hooks:
+    - id: check-signoff
+
+# Catch gofmt issues, if any.
+- repo: git://github.com/dnephin/pre-commit-golang
```

Comment:
can this skip commits that do not change `.go` files? Or is that automatic?

Reply:
It will get automatically skipped.